.github/workflows/master.yml

```diff
@@ -20,26 +20,27 @@ jobs:
         include:
           - java: 8
             spark: '3.0'
-            spark-hadoop: '2.7'
+            profiles: -Pspark-hadoop-2.7
           - java: 8
             spark: '3.1'
-            spark-hadoop: '3.2'
           - java: 8
             spark: '3.2'
-            spark-hadoop: '2.7'
-            profiles: '-Dspark.archive.mirror=https://archive.apache.org/dist/spark/spark-3.0.3 -Dspark.archive.name=spark-3.0.3-bin-hadoop2.7.tgz -Dmaven.plugin.scalatest.exclude.tags=org.apache.kyuubi.tags.ExtendedSQLTest,org.apache.kyuubi.tags.DeltaTest,org.apache.kyuubi.tags.HudiTest,org.apache.kyuubi.tags.IcebergTest'
+            profiles: >-
+              -Dspark.archive.mirror=https://archive.apache.org/dist/spark/spark-3.0.3
```

Comment:
i mean merging the following profiles as a list，not making it multi–line to look like a list